terraform/application/config/sandbox.tfvars.json

```diff
@@ -2,7 +2,7 @@
     "cluster": "production",
     "namespace": "bat-production",
     "statuscake_alerts": {
-        "website_url": [ "https://staging.claim-funding-for-mentor-training.education.gov.uk/healthcheck.json" ],
+        "website_url": [ "https://staging.claim-funding-for-mentor-training.education.gov.uk/healthcheck/all" ],
```

Comment:
this one should be sandbox.claim-funding-for-mentor-training.education.gov.uk/healthcheck/all (incorrect previously)